ocos/kernels/string_length.cc

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "string_length.hpp"
+#include "string_common.h"
+#include <vector>
+#include <locale>
+#include <codecvt>
+#include <algorithm>
+
+
+KernelStringLength::KernelStringLength(OrtApi api) : BaseKernel(api) {
+}
+
+void KernelStringLength::Compute(OrtKernelContext* context) {
+  // Setup inputs
+  const OrtValue* input = ort_.KernelContext_GetInput(context, 0);
+  std::vector<std::string> input_data;
+  GetTensorMutableDataString(api_, ort_, context, input, input_data);
```

Comment:
You should use wstring if you want to get the number of characters (see PR #65) otherwise it is the number of bytes.

Reply:
I got your concern, but wstring is encoding with UCS2 that doesn't support wider characters like emoji 😀.

Reply:
Don't use wstring, which is inconsistent between WIN32 and Linux. And more important, most of input is UTF-8 and the corresponding type is string, instead of wstring.